renderer/styles/components/activeProject.js

```diff
@@ -0,0 +1,21 @@
+export default `
+.activeProject {
+	padding-top:56px;
+}
+
+.mainTitle {
+	font-weight:normal;
+	font-size: 22px;
+	color: #7A848F;
+	margin-bottom:16px;
```

Comment:
Let op de spacing bij styles. margin-bottom: 16px; instead of margin-bottom:16px;. Zo houden we het consistent  😃

Reply:
Is goed 👍